packages/editor/src/components/widgets/code-editor/ScriptArea.tsx

```diff
@@ -6,7 +6,7 @@ import { usePath } from '../../../context';
 
 const ScriptArea = (props: CodeEditorAreaProps) => {
   const browser = useBrowser();
-  const { setEditor, modifyEditor } = useModifyEditor();
+  const { setEditor, modifyEditor } = useModifyEditor(undefined, true);
```

Comment:
normally I use an object if I have multiple optional attributes, so you only need to define the once you need.
e.g.:
```ts
export const useModifyEditor = (options?: {modifyAction?: ModifyAction, scriptAreaDatatypeEditor?: boolean})

useModifyEditor({scriptAreaDatatypeEditor: true});
```